CharacterCreation/03_Feats.txt

```diff
@@ -849,25 +848,25 @@ other weapons or class abilities. Does apply to Engineer mortars.
 
 == Melee Expertise ==
 
-	Applies to Melee Weapons that the user has Proficiency with and Unarmed Strike. 
+	Applies to Melee Weapons that the user has Proficiency with and to Unarmed Strike. 
 Does not apply to Improvised weapons. Reminder: these bonuses are only in effect while 
 you are wielding an appropriate weapon.
 
 0:	+1 Move Speed while using the Charge action with intent to engage an enemy in 
 	Melee Combat.
 12:	May use Weapons - Melee (Str/Dex) in place of Acrobatics or Athletics on 
-	maneuver checks to avoid, gain control of, or escape a Grapple; defend against 
-	Disarm attempts; or defend against Trip attempts.
+	maneuver checks to defend against Grapple, Disarm, or Trip attempts, or to gain 
+	control of or escape an ongoing Grappled state.
 24:	While wielding a Light or One-handed non-Hybrid weapon, whenever an enemy attacks 
 	you in Melee Combat and ties your Guard, you may make a melee attack against that 
 	enemy as a Reaction. 
-	While wielding a Hybrid or Two-handed weapon, you may choose to gain +2 to melee 
-	Accuracy checks for one round, but if you do then until the start of your next 
-	turn you take -2 to Guard checks.
-	If dual-wielding a Hybrid and Light or One-handed weapon, you may only use one 
-	of these effects per round, but may choose which.
-36:	+1 Accuracy when engaging a target wielding a melee weapon (unarmed counts).
-48:	+1 Guard when engaging a target wielding a melee weapon (unarmed counts).
+	While wielding a Hybrid or Two-handed weapon, at the start of each of your turns 
+	you may choose to gain +2 to melee Accuracy checks and take -2 to Guard checks 
+	until the start of your next turn. 
+	If dual-wielding a Hybrid weapon and a Light or One-handed weapon, you may only 
+	use one of these effects in a given round, but may choose which.
+36:	+1 Accuracy while engaging a target wielding a melee weapon (including unarmed targets).
+48:	+1 Guard while engaging a target wielding a melee weapon (including unarmed targets).

```

Comment:
I still think "while engaged with" leaves less  room for ambiguity than "while engaging." (the gerund can refer to either the process of entering engagement or to the entire engagement).